src/Logging/Logging.Console/ref/Microsoft.Extensions.Logging.Console.netcoreapp.cs

```diff
@@ -24,6 +24,7 @@ public partial class ConsoleLoggerOptions
         public bool IncludeScopes { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
         public Microsoft.Extensions.Logging.LogLevel LogToStandardErrorThreshold { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
         public string TimestampFormat { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
+        public bool UseUtcTimezone { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
```

Comment:
Time to open the naming thread :)

I think think should include some indication as to *where* we are using the timezone (i.e. the timestamp). Perhaps this:

```suggestion
        public bool UseUtcTimestamp { [System.Runtime.CompilerServices.CompilerGeneratedAttribute]get { throw null; } [System.Runtime.CompilerServices.CompilerGeneratedAttribute]set { } }
```

Reply:
cc @davidfowl @rynowak

Reply:
Give it's in `ConsoleLoggerOptions` I think it's pretty straightforward but happy to change it to `ConsoleLoggerOptions.UseUtcTimestampNotation`, `ConsoleLoggerOptions.UseUtcTimestampMessageNotation` or whatever you prefer, it's just a suggestion!

Reply:
I think `UseUtcTimezone` is sufficient here. I just think `UseUtcTimezone` wasn't entirely clear as to what we were affecting.

Reply:
`UseUtcTimestamp` is fine for me, ok for you @davidfowl & @rynowak ?

Reply:
👍

Reply:
Done!